dispatcher/brokers/pg_notify.py

```diff
@@ -63,6 +70,18 @@ def __init__(
         if not (config or sync_connection_factory or sync_connection):
             raise RuntimeError('Must specify either config or sync_connection_factory')
 
+        if not max_connection_idle_seconds:
+            raise RuntimeError('Must specify max_connection_idle_seconds')
```

Comment:
Let me formally describe a change I would like to make, just to add it to the list -

If `max_connection_idle_seconds` is None, I would like for this to signal "self-checks are disabled" as opposed to raising an exception. It's already `Optional`. I'm good with your default, but a user should be able to set to `None` and turn self-checks off.